### DISCUSSION TITLE
NET6 where is "Start External Program"?

### DISCUSSION THREAD

**Discussion by sunlight2424** [dotnet/wpf#7180] [Q&A] [answered]
Trying to migrate my WPF application from .net framework4.6.2 to NET6.0, it is a modularized application and has many plugins assembed with MEF.

In framework4.6.2, when debugging children module, it can be done by using "Start External Program" to start the main app and load current module. 
But in NET6.0 WPF application, there is no such option in "DEBUG" config. 

What can I do with this? Anything form commandline params?

Thanks!

**Comment by miloush** [comment, marked as answer]
Are you looking for the launch profiles under project properties > Debug > General?

![image](https://user-images.githubusercontent.com/10546952/194897638-d5e668fa-bcf8-4144-ad7e-6065b85a1f4e.png)

**Reply by sunlight2424** [reply]
Oh yes, Thanks!!
